tests/utils.py

```diff
@@ -13,17 +13,16 @@ def set_env():
 
 
 def more_than_two_gpus():
-    import pynvml
+    import numba.cuda
```

Comment:
Were we running into some issues using pynvml?

Reply:
I was just a bit awkward to use pynvml just to get a list of GPUs

Reply:
I think it's still better to use pynvml, even if the code is more cluttered. Using `numba.cuda` will enforce a context creation in all devices and pynvml won't.

Reply:
Thanks for the context. Yeah no strong feelings from me either way. It would be good to figure out how to make pynvml usage less awkward.

cc @rjzamora (in case you have thoughts on how we could improve this)